docs/quickstart.md

```diff
@@ -0,0 +1,51 @@
+# FastTrack Quickstart
+
+## Install Dependencies
+
+FastTrack requires the following dependencies to be installed on your system:
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Python 3](https://www.python.org/downloads/)
+
+```
+
+## Run FastTrack
+
+FastTrack can be run using the following command:
+
+```bash
+docker run --rm -p 5000:5000 -ti gresearch/fasttrack
+```
+
+Verify that you can see the UI by navigating to http://localhost:5000/.
+
+![FastTrack UI](https://files.mcaq.me/57b05.jpg)
```

Comment:
Could you include this file in your PR?
(Same remark for all the other screenshots)

Reply:
Ah. Yeah good point.